apps/web/src/services/iam/components/UserManagementTable.vue

```diff
@@ -302,11 +298,20 @@ const handleRemoveButton = async () => {
                                        :loading="dropdownState.loading"
                                        :search-text.sync="dropdownState.searchText"
                                        :handler="dropdownMenuHandler"
-                                       :disabled="!props.hasReadWriteAccess"
+                                       :disabled="!props.hasReadWriteAccess || isWorkspaceGroupUser(fieldItem)"
                                        class="role-select-dropdown"
                                        @select="handleSelectDropdownItem($event, rowIndex)"
                     >
                         <template #dropdown-button>
+                            <p-tooltip position="bottom"
+                                       :contents="useRoleFormatter(value?.type).name"
+                                       class="tooltip"
+                            >
+                                <img :src="useRoleFormatter(value?.type).image"
+                                     alt="role-type-icon"
```

Comment:
quibble: The `alt` attribute should contain easily readable text, something like "Role Type Icon"!

Reply:
GOOOOD, Thx U